### ISSUE TITLE
wrong-import-position error generated on PEP8 compliant imports w/ dunders

### ISSUE THREAD

**Issue by openfly** [PyCQA/pylint#965]
### Steps to reproduce
1. create python script like: https://gist.github.com/openfly/50ddd090a5ee8913dee945f932fe1906
2. run pylint --output-format=colorized --max-line-length=120 --disable=too-few-public-methods,no-member,too-many-ancestors example.py
3. see that there are errors:
C: 10, 0: Import "import sys" should be placed at the top of the module (wrong-import-position)
C: 11, 0: Import "import getopt" should be placed at the top of the module (wrong-import-position)
C: 12, 0: Import "import traceback" should be placed at the top of the module (wrong-import-position)

### Current behavior
Basically the dunder entries in the example.py above the imports are generating a hit on wrong-import-position

### Expected behavior
according to : https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names
this isn't actually a wrong position.  this is PEP8 compatible.

### pylint --version output
pylint --version
No config file found, using default configuration
pylint 1.5.6,
astroid 1.4.6
Python 2.7.11 (default, Jan 22 2016, 08:28:37)
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)]

**Comment by moylop260** [comment]
I got it
Thank you
I'll work this issue

**Comment by PCManticore** [comment]
Thank you @moylop260

**Comment by moylop260** [comment]
@openfly 
Could you review the following fix https://github.com/PyCQA/pylint/pull/966?
